### PR TITLE
DB-10139 Migrate Jenkins Env

### PR DESCRIPTION
**Pull request by bklo94**

### CODE REVIEW

**Comment by bklo94** [comment]
jenkinsbot branch cdh6.3.0

**Comment by bklo94** [comment]
jenkinsbot branch cdh6.3.0

**Comment by bklo94** [comment]
jenkinsbot branch cdh6.3.0

**Comment by bklo94** [comment]
jenkinsbot branch cdh6.3.0

**Comment by bklo94** [comment]
jenkinsbot branch cdh6.3.0

**Comment by bklo94** [comment]
jenkinsbot branch cdh6.3.0

**Comment by bklo94** [comment]
jenkinsbot branch cdh6.3.0

**Comment by bklo94** [comment]
jenkinsbot branch cdh6.3.0

**Comment by spliceci** [comment]
TEST

**Comment by bklo94** [comment]
jenkinsbot branch cdh6.3.0

**Comment by bklo94** [comment]
jenkinsbot branch cdh6.3.0

**Comment by cloudspliceci** [comment]
TRY AGAIN

**Comment by bklo94** [comment]
jenkinsbot branch @cdh6.3.0

**Comment by cloudspliceci** [comment]
TRY AGAIN

**Comment by bklo94** [comment]
jenkinsbot branch @cdh6.3.0

**Comment by cloudspliceci** [comment]
TRY AGAIN

**Comment by bklo94** [comment]
jenkinsbot branch @cdh6.3.0

**Comment by cloudspliceci** [comment]
TRY AGAIN

**Comment by bklo94** [comment]
jenkinsbot branch @cdh6.3.0

**Comment by cloudspliceci** [comment]
TRY AGAIN

**Comment by bklo94** [comment]
jenkinsbot branch @cdh6.3.0

**Comment by cloudspliceci** [comment]
TRY AGAIN

**Comment by bklo94** [comment]
jenkinsbot branch @hdp3.1.0

**Comment by cloudspliceci** [comment]
TRY AGAIN

**Comment by bklo94** [comment]
jenkinsbot branch @hdp3.1.0

**Comment by cloudspliceci** [comment]
TRY AGAIN

**Comment by bklo94** [comment]
jenkinsbot branch @hdp3.1.0

**Comment by cloudspliceci** [comment]
TRY AGAIN

**Comment by bklo94** [comment]
jenkinsbot branch @hdp3.1.0

**Comment by bklo94** [comment]
jenkinsbot branch @cdh6.3.0

**Comment by cloudspliceci** [comment]
TRY AGAIN

**Comment by cloudspliceci** [comment]
TRY AGAIN

**Review by ipraznik-splice — APPROVED** [review]

File: pipelines/spliceengine-PR/Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+                    sed  -i "s/REPLACE_PASS/$ARTIFACT_PASSWORD/" ~/.m2/settings.xml
+
+                    if [[ "\$platform" =~ ^mem ]] ; then
+                        profiles="core,mem"
+                    elif [[ "\$platform" =~ ^cdh ]] ; then
+                        profiles="core,ee,parcel,\$platform"
+                    elif [[ "\$platform" =~ ^hdp ]] ; then
+                        profiles="core,ee,hdp_service,\$platform"
+                    elif [[ "\$platform" =~ ^hdp ]] ; then
+                        profiles="core,ee,hdp_service,\$platform"
```
Comment (dgomezferro) [inline pipelines/spliceengine-PR/Jenkinsfile:85]: duplicated?

**Review by dgomezferro — APPROVED** [review]

**Review by carolp-503 — APPROVED** [review]
+1

**Pull request merged**